codegen.js

```diff
@@ -40,6 +40,11 @@ const codegen = async () => {
           ],
         },
       },
+      config: {
+        scalars: {
+          AWSDateTime: 'string',
```

Comment:
If we don't specify the config for a custom scalar, the generated type is any

Reply:
I think we use others like AWSEmail that could also be added here?